pkg/harvester/models/management.cattle.io.setting.js

```diff
@@ -27,6 +33,24 @@ export default class HciSetting extends HarvesterResource {
     return null;
   }
 
+  goToEdit(moreQuery = {}) {
+    const location = this.detailLocation;
+
+    if (location?.params?.id === 'ui-pl') {
```

Comment:
`ui-pl` should import from `pkg/harvester/config/settings.js`

Reply:
Done